webpack/components/RedHatRepositorySet/index.js

```diff
@@ -0,0 +1,64 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import cx from 'classnames';
+
+import { ListViewExpandableItem, ListViewItem } from 'patternfly-react';
+
+const RedHatRepositorySet = ({ redHatRepositorySet }) => {
+  const renderRepositoryList = () => (
+    redHatRepositorySet.repositories.map(redHatRepository =>
+      <ListViewItem key={redHatRepository.id} heading={redHatRepository.arch} />
+    )
+  );
+
+  // TODO: combine this and the one in RedHatRepository
```

Comment:
Is there something that will extract these TODOs? Or should these be Redmine issues to actually track them?